examples/time_series/goes_hek_m25.py

```diff
@@ -9,35 +9,34 @@
 
 from sunpy.net import Fido
 from sunpy.net import attrs as a
-from sunpy.net import hek
-from sunpy.time import TimeRange, parse_time
+from sunpy.time import parse_time
 from sunpy.timeseries import TimeSeries
 
 ###############################################################################
-# Let's first grab GOES XRS data for a particular time of interest
-tr = TimeRange(['2011-06-07 04:00', '2011-06-07 12:00'])
-results = Fido.search(a.Time(tr), a.Instrument.xrs)
+# Let's grab GOES XRS data for a particular time of interest and the HEK flare
+# data for this time from the NOAA Space Weather Prediction Center (SWPC)
+
+tr = a.Time('2011-06-07 04:00', '2011-06-07 12:00')
+results = Fido.search(tr, a.Instrument.xrs | a.hek.FL & (a.hek.FRM.Name == 'SWPC'))
 
 ###############################################################################
-# Then download the data and load it into a TimeSeries
+# Then download the XRS data and load it into a TimeSeries
 files = Fido.fetch(results)
 goes = TimeSeries(files)
 
 ###############################################################################
-# Next lets grab the HEK flare data for this time from the NOAA Space Weather
-# Prediction Center (SWPC)
-client = hek.HEKClient()
-flares_hek = client.search(hek.attrs.Time(tr.start, tr.end),
-                           hek.attrs.FL, hek.attrs.FRM.Name == 'SWPC')
+# Next lets load the HEK metadata from Fido result to a HEKRow
```

Comment:
```suggestion
# Next lets load the HEK metadata from Fido result to a HEKRow
```
This needs rewording. This will make no sense to most people.

Reply:
Reworded.